src/jobs/runAllJobs.ts

```diff
@@ -0,0 +1,51 @@
+import { Identity } from '@overture-stack/ego-token-middleware';
+import moment from 'moment';
+import { Transporter } from 'nodemailer';
+import SMTPTransport from 'nodemailer/lib/smtp-transport';
+
+import logger from '../logger';
+import attestationRequiredNotification from './attestationRequiredNotification';
+import runPauseAppsCheck from './pauseAppCheck';
+import { Report } from './types';
+import { getEmptyReport } from './utils';
+
+const JOB_NAME = 'ALL BATCH JOBS';
+
+export default async function (
+  emailClient: Transporter<SMTPTransport.SentMessageInfo>,
+  user: Identity,
+) {
+  logger.info(`${JOB_NAME} - Initiating...`);
+  // define currentDate here so each job has the same reference date
+  const currentDate = moment.utc().toDate();
+
+  try {
+    const attestationNotificationReport = await attestationRequiredNotification(
+      currentDate,
+      emailClient,
+    );
+    const pausedAppReport = await runPauseAppsCheck(currentDate, emailClient, user);
```

Comment:
These awaits result in the jobs running one at a time. If this is what you want then theres no harm. One job will run than another. You could just as easily do a Promise.allSettled on an array of your jobs and await the whole group to finish. Since a lot of your job's time is handling IO between database and email servers that might make the job faster.

Then again, we don't need the job to be faster ;)